{"name": "test_channel", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Swipe did not complete successfully", "trace": "self = <Script.test_channel_app.TestLogin object at 0x00C2BEF0>\n\n    def test_channel(self):\n        info = {\"username\": \"13922034602\", \"pwd\": \"10086\"}\n        time.sleep(1)\n        # 点击首页的我\n        self.page.get_page_home.click_me()\n        time.sleep(1)\n        # 点击用户中心的登陆\n        self.page.get_page_user_center.click_go_login()\n        time.sleep(1)\n        # 登陆\n>       self.page.get_page_login.login(info[\"username\"], info[\"pwd\"])\n\ntest_channel_app.py:39: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\Page_app\\page_login.py:71: in login\n    self.handle.write_password(password)\n..\\Page_app\\page_login.py:46: in write_password\n    self.input_text(self.Element.look_for_password(), password)\n..\\Page_app\\page_login.py:22: in look_for_password\n    return self.find_element_by_slide(self.password)\n..\\Base_app\\base_app.py:121: in find_element_by_slide\n    self.phone_slide(direction=direction)\n..\\Base_app\\base_app.py:97: in phone_slide\n    distance[1] * 0.6, duration=duration)\nF:\\MyProgram\\python36\\lib\\site-packages\\appium\\webdriver\\webdriver.py:508: in swipe\n    action.perform()\nF:\\MyProgram\\python36\\lib\\site-packages\\appium\\webdriver\\common\\touch_action.py:94: in perform\n    self._driver.execute(Command.TOUCH_ACTION, params)\nF:\\MyProgram\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:321: in execute\n    self.error_handler.check_response(response)\nF:\\MyProgram\\python36\\lib\\site-packages\\appium\\webdriver\\errorhandler.py:29: in check_response\n    raise wde\nF:\\MyProgram\\python36\\lib\\site-packages\\appium\\webdriver\\errorhandler.py:24: in check_response\n    super(MobileErrorHandler, self).check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <appium.webdriver.errorhandler.MobileErrorHandler object at 0x00C2BF90>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"An unknown server-side error occurred while pro...\\\\\\\\core-js\\\\\\\\modules\\\\\\\\_microtask.js:18:1)\\\\n    at process._tickCallback (internal/process/next_tick.js:61:11)\"}}'}\n\n    def check_response(self, response):\n        \"\"\"\n        Checks that a JSON response from the WebDriver does not have an error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get('status', None)\n        if status is None or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get('value', None)\n            if value_json and isinstance(value_json, basestring):\n                import json\n                try:\n                    value = json.loads(value_json)\n                    if len(value.keys()) == 1:\n                        value = value['value']\n                    status = value.get('error', None)\n                    if status is None:\n                        status = value[\"status\"]\n                        message = value[\"value\"]\n                        if not isinstance(message, basestring):\n                            value = message\n                            message = message.get('message')\n                    else:\n                        message = value.get('message', None)\n                except ValueError:\n                    pass\n    \n        exception_class = ErrorInResponseException\n        if status in ErrorCode.NO_SUCH_ELEMENT:\n            exception_class = NoSuchElementException\n        elif status in ErrorCode.NO_SUCH_FRAME:\n            exception_class = NoSuchFrameException\n        elif status in ErrorCode.NO_SUCH_WINDOW:\n            exception_class = NoSuchWindowException\n        elif status in ErrorCode.STALE_ELEMENT_REFERENCE:\n            exception_class = StaleElementReferenceException\n        elif status in ErrorCode.ELEMENT_NOT_VISIBLE:\n            exception_class = ElementNotVisibleException\n        elif status in ErrorCode.INVALID_ELEMENT_STATE:\n            exception_class = InvalidElementStateException\n        elif status in ErrorCode.INVALID_SELECTOR \\\n                or status in ErrorCode.INVALID_XPATH_SELECTOR \\\n                or status in ErrorCode.INVALID_XPATH_SELECTOR_RETURN_TYPER:\n            exception_class = InvalidSelectorException\n        elif status in ErrorCode.ELEMENT_IS_NOT_SELECTABLE:\n            exception_class = ElementNotSelectableException\n        elif status in ErrorCode.ELEMENT_NOT_INTERACTABLE:\n            exception_class = ElementNotInteractableException\n        elif status in ErrorCode.INVALID_COOKIE_DOMAIN:\n            exception_class = InvalidCookieDomainException\n        elif status in ErrorCode.UNABLE_TO_SET_COOKIE:\n            exception_class = UnableToSetCookieException\n        elif status in ErrorCode.TIMEOUT:\n            exception_class = TimeoutException\n        elif status in ErrorCode.SCRIPT_TIMEOUT:\n            exception_class = TimeoutException\n        elif status in ErrorCode.UNKNOWN_ERROR:\n            exception_class = WebDriverException\n        elif status in ErrorCode.UNEXPECTED_ALERT_OPEN:\n            exception_class = UnexpectedAlertPresentException\n        elif status in ErrorCode.NO_ALERT_OPEN:\n            exception_class = NoAlertPresentException\n        elif status in ErrorCode.IME_NOT_AVAILABLE:\n            exception_class = ImeNotAvailableException\n        elif status in ErrorCode.IME_ENGINE_ACTIVATION_FAILED:\n            exception_class = ImeActivationFailedException\n        elif status in ErrorCode.MOVE_TARGET_OUT_OF_BOUNDS:\n            exception_class = MoveTargetOutOfBoundsException\n        elif status in ErrorCode.JAVASCRIPT_ERROR:\n            exception_class = JavascriptException\n        elif status in ErrorCode.SESSION_NOT_CREATED:\n            exception_class = SessionNotCreatedException\n        elif status in ErrorCode.INVALID_ARGUMENT:\n            exception_class = InvalidArgumentException\n        elif status in ErrorCode.NO_SUCH_COOKIE:\n            exception_class = NoSuchCookieException\n        elif status in ErrorCode.UNABLE_TO_CAPTURE_SCREEN:\n            exception_class = ScreenshotException\n        elif status in ErrorCode.ELEMENT_CLICK_INTERCEPTED:\n            exception_class = ElementClickInterceptedException\n        elif status in ErrorCode.INSECURE_CERTIFICATE:\n            exception_class = InsecureCertificateException\n        elif status in ErrorCode.INVALID_COORDINATES:\n            exception_class = InvalidCoordinatesException\n        elif status in ErrorCode.INVALID_SESSION_ID:\n            exception_class = InvalidSessionIdException\n        elif status in ErrorCode.UNKNOWN_METHOD:\n            exception_class = UnknownMethodException\n        else:\n            exception_class = WebDriverException\n        if value == '' or value is None:\n            value = response['value']\n        if isinstance(value, basestring):\n            if exception_class == ErrorInResponseException:\n                raise exception_class(response, value)\n            raise exception_class(value)\n        if message == \"\" and 'message' in value:\n            message = value['message']\n    \n        screen = None\n        if 'screen' in value:\n            screen = value['screen']\n    \n        stacktrace = None\n        if 'stackTrace' in value and value['stackTrace']:\n            stacktrace = []\n            try:\n                for frame in value['stackTrace']:\n                    line = self._value_or_default(frame, 'lineNumber', '')\n                    file = self._value_or_default(frame, 'fileName', '<anonymous>')\n                    if line:\n                        file = \"%s:%s\" % (file, line)\n                    meth = self._value_or_default(frame, 'methodName', '<anonymous>')\n                    if 'className' in frame:\n                        meth = \"%s.%s\" % (frame['className'], meth)\n                    msg = \"    at %s (%s)\"\n                    msg = msg % (meth, file)\n                    stacktrace.append(msg)\n            except TypeError:\n                pass\n        if exception_class == ErrorInResponseException:\n            raise exception_class(response, message)\n        elif exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if 'data' in value:\n                alert_text = value['data'].get('text')\n            elif 'alert' in value:\n                alert_text = value['alert'].get('text')\n            raise exception_class(message, screen, stacktrace, alert_text)\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Swipe did not complete successfully\n\nF:\\MyProgram\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py:242: WebDriverException"}, "steps": [{"name": "点击首页中的我", "status": "passed", "start": 1591238321248, "stop": 1591238324705}, {"name": "点击登陆,跳转去登陆页面", "status": "passed", "start": 1591238329713, "stop": 1591238333563}, {"name": "登陆业务", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Swipe did not complete successfully\n\n", "trace": "  File \"F:\\MyProgram\\python36\\lib\\site-packages\\allure_commons\\_allure.py\", line 157, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\ASUS001\\Desktop\\GatherFrame\\Page_app\\page_login.py\", line 71, in login\n    self.handle.write_password(password)\n  File \"F:\\MyProgram\\python36\\lib\\site-packages\\allure_commons\\_allure.py\", line 157, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\ASUS001\\Desktop\\GatherFrame\\Page_app\\page_login.py\", line 46, in write_password\n    self.input_text(self.Element.look_for_password(), password)\n  File \"C:\\Users\\ASUS001\\Desktop\\GatherFrame\\Page_app\\page_login.py\", line 22, in look_for_password\n    return self.find_element_by_slide(self.password)\n  File \"C:\\Users\\ASUS001\\Desktop\\GatherFrame\\Base_app\\base_app.py\", line 121, in find_element_by_slide\n    self.phone_slide(direction=direction)\n  File \"C:\\Users\\ASUS001\\Desktop\\GatherFrame\\Base_app\\base_app.py\", line 97, in phone_slide\n    distance[1] * 0.6, duration=duration)\n  File \"F:\\MyProgram\\python36\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 508, in swipe\n    action.perform()\n  File \"F:\\MyProgram\\python36\\lib\\site-packages\\appium\\webdriver\\common\\touch_action.py\", line 94, in perform\n    self._driver.execute(Command.TOUCH_ACTION, params)\n  File \"F:\\MyProgram\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"F:\\MyProgram\\python36\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 29, in check_response\n    raise wde\n  File \"F:\\MyProgram\\python36\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 24, in check_response\n    super(MobileErrorHandler, self).check_response(response)\n  File \"F:\\MyProgram\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "输入用户名", "status": "passed", "attachments": [{"name": "13922034602", "source": "8f4887a8-dfbc-42de-9aef-6e7e1aff7305-attachment.attach"}], "parameters": [{"name": "username", "value": "'13922034602'"}], "start": 1591238338867, "stop": 1591238346690}, {"name": "点击手机空白的地方", "status": "passed", "start": 1591238346690, "stop": 1591238350441}, {"name": "输入密码", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Swipe did not complete successfully\n\n", "trace": "  File \"F:\\MyProgram\\python36\\lib\\site-packages\\allure_commons\\_allure.py\", line 157, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\ASUS001\\Desktop\\GatherFrame\\Page_app\\page_login.py\", line 46, in write_password\n    self.input_text(self.Element.look_for_password(), password)\n  File \"C:\\Users\\ASUS001\\Desktop\\GatherFrame\\Page_app\\page_login.py\", line 22, in look_for_password\n    return self.find_element_by_slide(self.password)\n  File \"C:\\Users\\ASUS001\\Desktop\\GatherFrame\\Base_app\\base_app.py\", line 121, in find_element_by_slide\n    self.phone_slide(direction=direction)\n  File \"C:\\Users\\ASUS001\\Desktop\\GatherFrame\\Base_app\\base_app.py\", line 97, in phone_slide\n    distance[1] * 0.6, duration=duration)\n  File \"F:\\MyProgram\\python36\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 508, in swipe\n    action.perform()\n  File \"F:\\MyProgram\\python36\\lib\\site-packages\\appium\\webdriver\\common\\touch_action.py\", line 94, in perform\n    self._driver.execute(Command.TOUCH_ACTION, params)\n  File \"F:\\MyProgram\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"F:\\MyProgram\\python36\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 29, in check_response\n    raise wde\n  File \"F:\\MyProgram\\python36\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 24, in check_response\n    super(MobileErrorHandler, self).check_response(response)\n  File \"F:\\MyProgram\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "attachments": [{"name": "10086", "source": "52705e6b-f4bb-4cb9-a146-e607348325c6-attachment.attach"}], "parameters": [{"name": "password", "value": "'10086'"}], "start": 1591238350442, "stop": 1591238425235}], "parameters": [{"name": "username", "value": "'13922034602'"}, {"name": "password", "value": "'10086'"}], "start": 1591238338867, "stop": 1591238425237}], "attachments": [{"name": "log", "source": "e261ed74-ca1c-4528-a3bb-785276adbe52-attachment.txt", "type": "text/plain"}], "start": 1591238316222, "stop": 1591238425238, "uuid": "08619cc2-1563-44eb-9846-fe2d64a7ab21", "historyId": "9322ae435828398b7c90b122fa6c93b7", "testCaseId": "396a6fd21872a2874f2e97c367a88a98", "fullName": "Script.test_channel_app.TestLogin#test_channel", "labels": [{"name": "parentSuite", "value": "Script"}, {"name": "suite", "value": "test_channel_app"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-03F992M"}, {"name": "thread", "value": "1256-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Script.test_channel_app"}]}